src/styles/typography.scss

```diff
@@ -1,3 +1,5 @@
+@use 'variables';
```

Comment:
> Module Warning (from ./node_modules/sass-loader/dist/cjs.js):
Deprecation Warning on line 8, column 8 of file:///Users/denis/Documents/superhero-wallet/src/styles/typography.scss:8:8:
Sass @ import rules are deprecated and will be removed in Dart Sass 3.0.0.
More info and automated migrator: https://sass-lang.com/d/import